check_version.sh

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+set -eu
+
+# VARIABLES TO CHANGE
+library="java"
```

Comment:
These currently need to be edited manually. But I think this is useful enough in its current form to go ahead with it. Then someone might want to refine.